src/ostorlab/runtimes/local/runtime.py

```diff
@@ -309,10 +309,10 @@ def stop(self, scan_id: str) -> None:
         networks = self._docker_client.networks.list()
         for network in networks:
             network_labels = network.attrs["Labels"]
-            if (
-                network_labels is not None
-                and int(network_labels.get("ostorlab.universe")) == scan_id
-            ):
+            if network_labels is None:
```

Comment:
I would use .get() & default to an empty dictionary to avoid unnecessary iindentation..

Reply:
+1

Reply:
Not a safe default. None indicates absence, empty dict indicates presence and missing value.